src/main/java/mekanism/common/CommonPlayerTickHandler.java

```diff
@@ -382,4 +386,21 @@ private FallEnergyInfo getFallAbsorptionEnergyInfo(LivingEntity base) {
 
     private record FallEnergyInfo(@Nullable IEnergyContainer container, FloatSupplier damageRatio, FloatingLongSupplier energyCost) {
     }
+
+    @SubscribeEvent
+    public void getBreakSpeed(BreakSpeed event) {
+        Player player = event.getPlayer();
+        IModule<ModuleGyroscopicStabilizationUnit> module = MekanismAPI.getModuleHelper().load(player.getItemBySlot(EquipmentSlot.CHEST), MekanismModules.GYROSCOPIC_STABILIZATION_UNIT);
+        if (module != null && module.isEnabled()) {
+            float speed = event.getOriginalSpeed();
```

Comment:
Do we maybe want to compare original and new speed to estimate if another mod already is doing this? (Such as if new is 5x original just don't do anything?)

Also I feel like if our calculated speed is less than the new speed we may not want to be updating the break speed. Though maybe we should just be operating on the new speed in the first place? That way we don't cause some other thing from mods to get overridden because they happened before us.

Reply:
I don't really like the idea of comparing new and original to figure out if they've already done it as they might get called after us, but if they also do that it might work. I could see just using the new speed, it might result in really fast break speeds but only when you're in the air... and most likely with a tool which can have 128 efficiency anyway, because of the edge case nature I think I'm honestly ok with whatever.